ci/builders/linux_fuchsia.json

```diff
@@ -86,6 +89,15 @@
                         "${REVISION}",
                         "--upload"
                     ]
+                },
+                {
+                    "name": "arm64 emulator based debug tests",
```

Comment:
With this change, are we able to delete "Linux Fuchsia FEMU" and "Linux Fuchsia arm64 FEMU" from [ci.yaml](https://github.com/flutter/engine/blob/main/.ci.yaml)?

Reply:
We are still missing `dart-aot-runner-integration-test` in `fuchsia_profile_arm64`. It's same as the `fuchsia_profile_x64`, needs `--no-lto` to avoid timing out. But I need to confirm if `fuchsia_profile_arm64` is used in smart display product.